ompi/mca/coll/libnbc/coll_libnbc_component.c

```diff
@@ -158,6 +158,15 @@ libnbc_register(void)
                                            MCA_BASE_VAR_SCOPE_READONLY,
                                            &libnbc_ibcast_skip_dt_decision);
 
+    libnbc_iscan_algorithm = 0;
+    (void) mca_base_component_var_register(&mca_coll_libnbc_component.super.collm_version,
```

Comment:
We should probably at some point add an enumerator for the algorithms so they can be selected by name.